docs/src/tutorials/weak_order_conv_sde.md

```diff
@@ -0,0 +1,41 @@
+# [Using the EnsembleGPUKernel SDE solvers for the expectation of SDEs ](@id sdeweakconv)
+
+Solving the `SDEProblem` using weak methods on multiple trajectories helps to generate the expectation of the stochastic process.
+With the lower overhead of `EnsembleGPUKernel` API, these calculations can be done in parallel on GPU, potentially being faster.
+
+The example below provides a way to calculate the expectation time-series of a linear SDE:
+
+```julia
+using DiffEqGPU, OrdinaryDiffEq, StaticArrays, LinearAlgebra, Statistics
+
+num_trajectories = 10_000
+
+# Defining the Problem
+# dX = pudt + qudW
+u₀ = SA[0.1f0]
+f(u, p, t) = SA[p[1] * u[1]]
+g(u, p, t) = SA[p[2] * u[1]]
+tspan = (0.0f0, 1.0f0)
+p = SA[1.5f0, 0.01f0]
+
+prob = SDEProblem(f, g, u₀, tspan, p; seed = 1234)
+
+monteprob = EnsembleProblem(prob)
+
+sol = solve(monteprob, GPUEM(), EnsembleGPUKernel(0.0), dt =  Float32(1//2^8),
+            trajectories = num_trajectories, adaptive = false)
+        
+sol_array = Array(sol)
+
+ts = sol[1].t
+
+us_calc = reshape(mean(sol_array, dims = 3), size(sol_array, 2))
+
+us_exact = u₀ .* exp.(p[1] * ts)
```

Comment:
```suggestion
us_exact = u₀ .* exp.((p[1] - 0.5* p[2]^2)  * ts)
```

Reply:
Why was this removed?

Reply:
Nevermind: https://en.wikipedia.org/wiki/Geometric_Brownian_motion it is correct.

Reply:
Yeah, E[exp^(p[2]*W)] is exp^(p[2]^/2) which cancels the Ito's correction term.